src/styles.css

```
@@ -1430,6 +1454,53 @@ a:link {
     }
 }
 
+#minimap .boulder,
+.mapCellDetails .enlargedTile.boulder {
+    position: relative;
+
+    &::before {
+        position: absolute;
+        left: calc(50% - (1ch / 2));
+        top: 0;
+        width: 1ch;
+        overflow: hidden;
+        content: " ";
+        display: inline-block;
+        animation: boulderRoll 0.6s infinite;
+        color: #8B5A28;
+    }
+}
+
+.mapCellDetails .enlargedTile.boulder::before {
+    top: 0.4ch !important;
+}
+
+#minimap .boulder {
+    width: 1ch;
+}
+
+@keyframes boulderRoll {
+    0%, 25% {
+        content: "o";
+        transform: scaleY(1);
+    }
+
+    25.01%, 50% {
+        content: "o";
+        transform: scaleY(-1);
+    }
+
+    50.01%, 75% {
+        content: "o";
+        transform: scaleY(1);
+    }
+
+    75.01%, 100% {
+        content: "o";
+        transform: scaleY(-1);
+    }
+}
```

Since the `content` never changes, these could technically be placed [within the pseudoelement directly](https://github.com/packardbell95/tardquest/pull/175/files#diff-4753b2e6427841425517139f09f1342c320338443447af46169dc0f43a0f2cf3R1461) rather than included in each keyframe. Though this is minimal and won't change any current behavior

The animation itself also flips the "o" upside down twice with an equal duration (0-50%, then 50.01-100%). This could be reduced with a jumping animation too:



```suggestion
        content: "o";
        display: inline-block;
        animation: boulderRoll 0.3s steps(2, end) infinite;
        color: #8B5A28;
    }
}

.mapCellDetails .enlargedTile.boulder::before {
    top: 0.4ch !important;
}

#minimap .boulder {
    width: 1ch;
}

@keyframes boulderRoll {
    0%, 50% {
        transform: scaleY(1);
    }

    50.01%, 100% {
        transform: scaleY(-1);
    }
}
```

(I think GitHub's multi-line diff is broken, so this might not show up as nicely as it should)